pt-BR/mail/chrome/messenger/addressbook/abMainWindow.dtd

```diff
@@ -200,15 +200,15 @@
 <!ENTITY _PhoneticName.accesskey                        "f">
 
 <!-- Results pane context menu -->
-<!ENTITY composeEmail.label                             "Compose Email To">
+<!ENTITY composeEmail.label                             "Criar email para">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY composeEmail.label                             "Criar mensagem de correio eletrônico para">